src/toast/ops/noise_estimation.py

```diff
@@ -128,7 +128,8 @@ class NoiseEstim(Operator):
     )
 
     output_dir = Unicode(
-        ".",
+        None,
```

Comment:
Noise estimation code in the WCS PR?

Reply:
These are residual small things that were missed in #571 .  Apologies to slip them in here.